tests/test_other.py

```diff
@@ -2399,6 +2399,10 @@ def test_embind(self):
       if fail:
         self.assertNotEqual(proc.returncode, 0)
       else:
+        if 'DYNAMIC_EXECUTION=0' in args:
+          output = run_js(self.in_dir('a.out.js'), stdout=PIPE, stderr=PIPE, assert_returncode=0, engine=NODE_JS,
+                          shell_args=['--disallow-code-generation-from-strings'])
```

Comment:
Looks like this test fails on CI, but it passes for me locally. Perhaps this flag is just present in some versions of node.js?

Reply:
It may be safer to just scan the output for `eval(` and `new Function`.

Reply:
Yes, running locally with Node 8.9.1 (the version used by CircleCI) produces:

/usr/local/google/home/wfurr/Projects/node-v8.9.1-linux-x64/bin/node: bad option: --disallow-code-generation-from-strings

Reply:
Changed to string check output per your suggestion.  It has to be "new Function(" because "new Function" appears in an error message: https://github.com/kripken/emscripten/blob/incoming/src/embind/embind.js#L852